packages/inventory/src/Stocktake/ListView/CreateStocktakeButton.tsx

```diff
@@ -8,15 +8,15 @@ import { StockItemSelectModal } from 'packages/system/src';
 
 export const CreateStocktakeButton: React.FC = () => {
   const t = useTranslation(['distribution', 'common']);
-  const { mutate } = useInsertStocktake();
+  const { mutateAsync } = useInsertStocktake();
   const modalController = useToggle();
 
   return (
     <>
       {modalController.isOn && (
         <StockItemSelectModal
           isOpen={modalController.isOn}
-          onChange={mutate}
+          onChange={mutateAsync}
```

Comment:
Just so that I can `await` it correctly - no impact though, as the `onSuccess` is navigating anyway